src/js/components/tour/dropdown.tsx

```diff
@@ -61,7 +61,7 @@ const TourDropdown = ({ isAlert }: { isAlert?: boolean }) => {
   );
 
   return project ? (
-    <div className="slds-col_padded">
+    <>
```

Comment:
Interesting; was the padding not needed here? If so, we can remove this `<>...</>` entirely since it's just one node.

Reply:
yeah, I thought it was fine without the padding. and without the `div` the dropdown was already displaying inline.